tasks/util/constants.js

```diff
@@ -12,8 +12,10 @@ var pathToBuild = path.join(pathToRoot, 'build/');
 
 var pathToPlotlyIndex = path.join(pathToLib, 'index.js');
 var mainIndex = fs.readFileSync(pathToPlotlyIndex, 'utf-8');
-var pathToPlotlyTraces = path.join(pathToSrc, 'traces');
-var allTraces = fs.readdirSync(pathToPlotlyTraces);
+var allTraces = fs.readdirSync(path.join(pathToSrc, 'traces'));
+var allTransforms = fs.readdirSync(path.join(pathToSrc, 'transforms'))
+    .map(function(e) { return e.replace('.js', ''); });
+allTransforms.splice(allTransforms.indexOf('helpers'), 1);
```

Comment:
```suggestion
var allTraces = fs.readdirSync(path.join(pathToSrc, 'traces'))
    .filter(function(v) {
        return v.charAt(0) !== v.charAt(0).toUpperCase();
    });
var allTransforms = fs.readdirSync(path.join(pathToSrc, 'transforms'))
    .filter(function(v) {
        return v.charAt(0) !== v.charAt(0).toUpperCase() && v !== 'helpers.js';
    })
    .map(function(e) { return e.replace('.js', ''); });
```

Only include files that start with a lowercase letter. Specifically my problem running this on a mac was:

```
Unable to find and drop require for .DS_Store
```

But there could easily be more `.`, `~`, other system-generated files we want to ignore.